tests/application/conftest.py

```diff
@@ -27,13 +26,11 @@
 
 REWARD_DURATION = 60 * 60 * 24 * 7  # one week in seconds
 DEAUTHORIZATION_DURATION = 60 * 60 * 24 * 60  # 60 days in seconds
-TOTAL_SUPPLY = Web3.to_wei(1_000_000_000, "ether")  # TODO NU(1_000_000_000, 'NU').to_units()
+TOTAL_SUPPLY = Web3.to_wei(10_000_000_000, "ether")  # TODO NU(10_000_000_000, 'NU').to_units()
```

Comment:
If defining here then remove L25 - that's what influenced the testing confusion.

Reply:
Side note, there is going to be a mint before TACo launch that could push the total supply closer to 11B, perhaps we should use that value.

Reply:
11B comparing to 10B is not important because main thing is to catch order of values

Reply:
✔️